### PR TITLE
Fixing the make run

### PR DESCRIPTION
**Pull request by mchernyavskaya**
Fixing `make run` to take the service type from app config instead of config in kibana.

#### Pre-Review Checklist
- [ ] Covered the changes with automated tests
- [x] Tested the changes locally

### CODE REVIEW

**Review by artem-shelkovnikov — APPROVED** [review]

**Comment by vidok** [comment]
@mchernyavskaya I noticed there might be something wrong with your local `main` branch. I see you changed only 2 files but GitHub shows a lot of commits. Also [here](https://github.com/elastic/connectors/commit/076c608015278b0a3cd4c17d3a0a8d92635b4bec). I think everything is okay in terms of code however, it made commit description a bit hard to follow.

**Comment by mchernyavskaya** [comment]
replaced by https://github.com/elastic/connectors/pull/149 to fix git history issues pointed out by @vidok
